cluster/gce/util.sh

```diff
@@ -1737,6 +1946,15 @@ function create-node-template() {
     "${ENABLE_IP_ALIASES:-}" \
     "${IP_ALIAS_SIZE:-}")
 
+  local node_image_flags=""
+  if [[ "${linux_or_windows}" == 'linux' ]]; then
+      node_image_flags="--image-project ${LINUX_NODE_IMAGE_PROJECT} --image ${LINUX_NODE_IMAGE}"
+  else
```

Comment:
it's a string not a boolean let's make sure it's "windows"

Reply:
Done.